daemon/health.go

```diff
@@ -156,6 +157,7 @@ func monitor(d *Daemon, c *container.Container, stop chan struct{}, probe probe)
 			ctx, cancelProbe := context.WithTimeout(context.Background(), probeTimeout)
 			results := make(chan *types.HealthcheckResult)
 			go func() {
+				healthCheckCounter.Inc()
```

Comment:
The end of `handleProbeResult` seems like a good spot. 

Can add 
```
if h.Status == types.Healthy {
    healthyCheckCounter.Inc()
} else {
    unhealthyCheckCounter.Inc()
}
```
to the end of the method.